### ISSUE TITLE
actor loss function:reduce_sum() or mean()

### ISSUE THREAD

**Issue by lwz339** [godka/Pensieve-PPO#34]
I have a problem in the loss function.I found in the pytorch version,the actor loss function is defined as :
`loss = -dual_loss.mean() + 10. * F.mse_loss(val, v_batch) - self._entropy_weight * loss_entropy.mean()`
and in the tensorflow version:
`self.loss = - tf.reduce_sum(self.dual_loss) \
            + self.entropy_weight * tf.reduce_sum(self.entropy)`.
And I found that in critic is always MSE loss.
Could you please tell me how did you choose the two implements？Or it's not a big problem to choose one of them?
Thank you very much!

**Comment by godka** [comment]
Hi, 

Nice catch! Actually, in the Torch version, I merged the actor and critic loss into ``one'' optimization step -- that’s why I used `.mean()` to normalize the loss. By contrast, in the TF version, I updated the actor and critic separately, so using `tf.reduce_sum` there doesn’t make too much difference in practice --  I mean, just scaling with the learning rate I guess....

Best,
Tianchi

**Comment by lwz339** [comment]
Your clarification is very clear.
This repo taught me a lot.Thanks for your contribution!
